packages/reporter/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@lage-run/reporter",
+  "version": "1.1.0",
+  "description": "Log reporters for Lage",
+  "repository": {
+    "url": "https://github.com/microsoft/lage"
+  },
+  "license": "MIT",
+  "main": "lib/index.js",
```

Comment:
i don't see `src/index.*` in this PR, missing?

Reply:
ahhhh crap, always forget. ain't no unit test gonna test that :(